--- _resources/port1.0/fetch/mirror_sites.tcl.orig	2026-07-24 16:59:17.000000000 -0600
+++ _resources/port1.0/fetch/mirror_sites.tcl	2026-07-24 16:59:57.000000000 -0600
@@ -377,8 +377,10 @@
 variable ykf.ca      http
 variable fcix.net    http
 variable sjtu.edu.cn ${letsencrypt_https_only}
+variable tigerports.com http
 
 set sites(macports_distfiles) [lsearch -all -glob -inline -not "
+    ${tigerports.com}://tigerports.com/macports/distfiles/:mirror
     ${fastly}://distfiles.macports.org/:mirror
     ${nue.de}://nue.de.distfiles.macports.org/:mirror
     ${fcix.net}://mirror.fcix.net/macports/distfiles/:mirror
