--- scipy/meson.build	2026-06-19 11:00:59.000000000 +0800
+++ scipy/meson.build	2026-07-04 19:17:10.000000000 +0800
@@ -265,9 +265,14 @@
 
 macOS13_3_or_later = false
 if host_machine.system() == 'darwin'
-  r = run_command('xcrun', '-sdk', 'macosx', '--show-sdk-version', check: true)
-  sdkVersion = r.stdout().strip()
-  macOS13_3_or_later = sdkVersion.version_compare('>=13.3')
+  r = run_command('xcrun', '-sdk', 'macosx', '--show-sdk-version', check: false)
+  errortxt = r.stderr().strip()
+  if errortxt != ''
+      macOS13_3_or_later = false
+  else
+      sdkVersion = r.stdout().strip()
+      macOS13_3_or_later = sdkVersion.version_compare('>=13.3')
+  endif
 endif
 
 _args_blas_lp64 = []
