--- lib/x86/decompress_impl.h.orig	2026-08-15 00:39:16.000000000 -0600
+++ lib/x86/decompress_impl.h	2026-08-15 00:40:00.000000000 -0600
@@ -16,6 +16,7 @@
  * intrinsics.  It seems to be fixed in VS2022.  Hence, use MSVC_PREREQ(1930).
  */
 #if defined(__GNUC__) || defined(__clang__) || MSVC_PREREQ(1930)
+#if !defined(LIBDEFLATE_ASSEMBLER_DOES_NOT_SUPPORT_BMI2) 
 #  define deflate_decompress_bmi2	deflate_decompress_bmi2
 #  define FUNCNAME			deflate_decompress_bmi2
 #  define ATTRIBUTES			_target_attribute("bmi2")
@@ -38,6 +39,7 @@
 #  endif
 #  include "../decompress_template.h"
 #endif
+#endif
 
 #if defined(deflate_decompress_bmi2) && HAVE_BMI2_NATIVE
 #define DEFAULT_IMPL	deflate_decompress_bmi2
